models/branches.go

```diff
@@ -596,3 +596,100 @@ func RemoveOldDeletedBranches(ctx context.Context, olderThan time.Duration) {
 		log.Error("DeletedBranchesCleanup: %v", err)
 	}
 }
+
+// RenamedBranch proivde renamed branch log
+// will check it when an branch can't be found
+type RenamedBranch struct {
+	ID     int64 `xorm:"pk autoincr"`
+	RepoID int64 `xorm:"INDEX NOT NULL"`
+	From   string
+	To     string
+}
```

Comment:
can you add a time stamp?

```go
CreatedUnix      timeutil.TimeStamp `xorm:"INDEX created"`
```

Reply:
I don't think it's usefull...

Reply:
I'm having some prune after time X for admins in mind ...